### ISSUE TITLE
Why is the voice cloned on the Resemble.AI. platform better than this project?

### ISSUE THREAD

**Issue by wyp19930313** [CorentinJ/Real-Time-Voice-Cloning#590]
When I cloned my voice on the Resemble.AI. platform, it was very similar to the real voice. But when I clone my voice with this project, the similarity is very low. Do they use the same model? If the same model is used, is the training data different? Or the platform finetune the modle to inference?

**Comment by wyp19930313** [comment]
Look forward to your answer, thanks.

**Comment by deltabravozulu** [comment]
I believe resemble.ai uses distributed cloud buckets to store and iterate on user-specific models. That is, *I think* it has a huge training set for its base models, then specifically trains account-centric models separately using a much stronger set of computers than a normal human being would be able to afford. I might be wrong, of course.

**Comment by mbdash** [comment]
See @blue-fish experiments here:
https://github.com/CorentinJ/Real-Time-Voice-Cloning/issues/437

**Comment by CorentinJ** [comment]
The short answer: it's an industry-grade project.

There's 5 machine learning engineers (myself included) and 4 non-ML engineers in the Resemble team. We've been iterating over the project for 2 years and we have contracts with other companies that are expecting strong results from us. We're far away from this student project which I built alone in 8 months while studying other courses; even though ML has its way of making every small incremental improvement harder to get than the last.

Our TTS engine at Resemble contains hundreds of improvements over this one, as well as a truckload of features that allow for more applications and more customization. We also indeed have a different routine for cloning a voice than what is done here. SV2TTS has its limitations (as noted by the authors themselves) and while it serves a nice purpose, we are aiming for a higher overall synthesis quality. What @mbdash suggests should already be a good step forward.
